src/components/DexToken.vue

```diff
@@ -0,0 +1,84 @@
+<script setup lang="ts">
+import { DEX_TOKEN, DEX_TOKEN_DECIMALS, DEX_TOKEN_REFETCH_INTERVAL, CurrencySymbol, CURRENCY_USD } from '@/core'
+import { TokenAmount, Price, TokenImpl } from '@/core/entities'
+import { useTokensQuery } from '@/query/tokens-derived-usd'
+import BigNumber from 'bignumber.js'
+
+const dexStore = useDexStore()
+
+const dexToken = new TokenImpl({
+  address: DEX_TOKEN,
+  decimals: DEX_TOKEN_DECIMALS,
+  symbol: 'DEX' as CurrencySymbol,
+  name: 'DEX Token',
+})
+
+const TokensQuery = useTokensQuery([dexToken.address], { pollInterval: DEX_TOKEN_REFETCH_INTERVAL })
+TokensQuery.load()
```

Comment:
Actually, it is better to include `DEX` token as a special token into the tokens store. It already has a lot of logic and allows to not duplicate logic here.